src/coverlet.core/Instrumentation/ReachabilityHelper.cs

```diff
@@ -169,7 +169,9 @@ public override string ToString()
         /// OpCodes that transfer control code, even if they do not
         /// introduce branch points.
         /// </summary>
+#pragma warning disable IDE1006 // Naming Styles
```

Comment:
I thought you would prefer not to change the name of the field but if you are ok with changing, please let me know.

Reply:
I'm ok with field name update